app/views/page/show.html.erb

```diff
@@ -175,7 +175,24 @@
             <% end %>
           </div>
 
-        <%# Text and Images %>
+          <%# Map Component %>
+        <% when 'PageMapComponent' %>
+          <%
+            description = component.description
+            title = component.title
+            next_component = @page.page_components.find_by(position: pc.position + 1)
+          %>
+          <div class="margin-bottom-2 font-sans-lg text-bold margin-top-9">
+            <%= content_tag(:span, title, class: 'margin-right-1') if title != '' %>
```

Comment:
non-blocking tip: you can also use `title.present?` for checking whether variables are `nil` or empty strings.

Reply:
yeah.. i agree.  Done.